bundles/admin/admin-layereditor/view/AdminLayerForm/AdditionalTabPane/SingleTile.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Switch, Message } from 'oskari-ui';
+import { LocaleConsumer, Controller } from 'oskari-ui/util';
+import { InfoTooltip } from '../InfoTooltip';
+import { StyledFormField } from '../styled';
+import styled from 'styled-components';
+
+const SpacedLabel = styled('div')`
+    display: inline-block;
+    margin-left: 10px;
+`;
+
+export const SingleTile = LocaleConsumer(({ layer, controller }) => {
```

Comment:
I think this doesn't need to be wrapped in LocaleConsumer for it to work. You only need LocaleConsumer if you need the getMessage() function but since this uses Message-tag (like it should) this should work without the wrapper.